lib/url.c

```diff
@@ -1086,8 +1086,8 @@ ConnectionExists(struct Curl_easy *data,
         /* connect-only or to-be-closed connections will not be reused */
         continue;
 
-      multiplexed = CONN_INUSE(check) &&
-        (bundle->multiuse == BUNDLE_MULTIPLEX);
+      if(bundle->multiuse == BUNDLE_MULTIPLEX)
+        multiplexed = CONN_INUSE(check);
```

Comment:
This changes it from true/false to the count of handles. I suggest check all evaluations of this variable.

Reply:
`multiplexed` is a `size_t` already since it was always a counter before this regression. Did you see any particular reason this is now a problem?

Reply:
> `multiplexed` is a `size_t` already since it was always a counter before this regression. Did you see any particular reason this is now a problem?

I see. multiplexed is meant to be a count of handles but was erroneously set true/false. So instead I suggest put that in the commit message, it is not obvious from the change.

https://github.com/curl/curl/blob/2e9b725f67d49a9d7a1f053fe52dd4920c9ab1ad/lib/url.c#L1343-L1354